.github/workflows/docker-test.yml

```diff
@@ -54,6 +54,18 @@ jobs:
           echo DOCKER_TOKEN=${{ secrets.DOCKER_TOKEN }} >> ${ENV_FILE_PATH}
           cat ${ENV_FILE_PATH}
 
+      - name: 'Get Server version'
+        run: |
+          DOCKER_IMAGE_SERVER=$(npm pkg get version --prefix ./apps/server | xargs)
+          echo DOCKER_IMAGE_SERVER="${DOCKER_IMAGE_SERVER}" >> ./.env
+          echo DOCKER_IMAGE_SERVER="${DOCKER_IMAGE_SERVER}" >> $GITHUB_ENV
+
+      - name: 'Get UI version'
+        run: |
+          DOCKER_IMAGE_UI=$(npm pkg get version --prefix ./apps/ui | xargs)
+          echo DOCKER_IMAGE_UI="${DOCKER_IMAGE_UI}" >> ./.env
+          echo DOCKER_IMAGE_UI="${DOCKER_IMAGE_UI}" >> $GITHUB_ENV
+
```

Comment:
Code that was should be here. While it doesn't cause issues if it's missing, it is a better mirror of the actual docker build process if this code is here.